### ISSUE TITLE
Colour palette experiments

### ISSUE THREAD

**Issue by aditiagrwl** [batuaWallet/ux#11]
![Colour palette experiments](https://user-images.githubusercontent.com/72272139/95656388-b00be180-0b2b-11eb-9749-8340a683d6ec.png)

**Comment by aditiagrwl** [comment]
Some colour trails. Yet to experiment with more. Inclined towards the first palette though.

**Comment by chrishobcroft** [comment]
Agreed, the first palette feels professional yet warm.

The red and yellow appealed initially - reminds me of the coloured powder sold by street vendors around Holi time... but on further looking, they start to feel blunt and even clumsy :)

**Comment by aditiagrwl** [comment]
Yeah I was just playing around with colours. I like the first one too. Trying some gradients around it too!

**Comment by bohendo** [comment]
First color palette looks great to me.

I'd be curious to see an example in orange & green too tho :grin: :india:
